packages/twenty-server/src/engine/metadata-modules/utils/validate-string-input.utils.ts

```diff
@@ -0,0 +1,7 @@
+export const INPUT_REGEX = /^([A-Za-z0-9\-_.@]+)$/;
+
+export const validateStringInput = (input: string) => {
```

Comment:
why did you put this function outside of the remote server util? The regex is really specific to remote servers. Unless you send the regex as input